packages/telemed-ehr/app/src/pages/Schedule.tsx

```diff
@@ -147,6 +147,19 @@ export default function SchedulePage(): ReactElement {
         valueString: START_SCHEDULE,
       },
     ];
+
+    // if there is no timezone extension, add it. The default timezone is America/New_York
+    if (
+      !item?.extension?.find((ext) => {
+        return ext.url === 'http://hl7.org/fhir/StructureDefinition/timezone';
+      })
+    ) {
+      scheduleExtension.push({
+        url: 'http://hl7.org/fhir/StructureDefinition/timezone',
```

Comment:
Would be nice to move this link to a constant and use it, because it has been used several times in the code